style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+#wrap {
+  width: 375px;
+  height: 638px;
+}
+
+.nav-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 375px;
+  height: 48px;
+  background-color: #3c3a39;
+}
+
+#logo {
+  display: flex;
+  align-items: center;
+  width: 77px;
+  height: 24px;
+  margin-left: 12px;
+  padding: 10px 12px;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 40px;
+  height: 40px;
+  border-radius: 8px;
+  background-color: #3c3a39;
+  border: none;
+}
+
+.button:hover {
+  background-color: #fff5e1;
+}
+
+#nav-icon {
+  margin-right: 24px;
+}
+
+#container {
+  display: flex;
+  background-color: black;
+  background-image: url(/bacground-icons/background-image.png);
+  width: 375px;
+  height: 590px;
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
```

Comment:
- [ ] Dear @vikas924, Please consider using the appropriate background color `color: #1C1A19` for this section. I notice you apply a black, but the figma design uses `#1C1A19`. This is to align your design with the figma design better.
- [ ] Also The background image is not visible. I suspect this is because of how you refer to the image. I suggest you use the `./` to tell the browser it's a relative path. something like `URL("./bacground-icons/background-image.png")`

### Figma design 
![Screenshot 2023-02-16 172322](https://user-images.githubusercontent.com/69077061/219392394-6918a362-f8b7-461c-913a-f57fbbd7ab3c.png)

### Your design 
![Screenshot 2023-02-16 172243](https://user-images.githubusercontent.com/69077061/219391971-a7a7a485-4c3d-490d-aff6-b5b0b20f3069.png)